poucave/html/css/svg-diagram.css

```diff
@@ -0,0 +1,69 @@
+@import url(https://fonts.googleapis.com/css?family=Source+Sans+Pro);
```

Comment:
I'm not a fan of pulling stuff from Google TBH
Can we serve the woff files ourselves?